deployment/ansible/roles/model-my-watershed.app/defaults/main.yml

```diff
@@ -1,26 +1,17 @@
 ---
 app_home: /opt/app
-app_config_home: /etc/mmw.d/env
 app_deploy_branch: "master"
 
 app_config:
-  - { file: "MMW_DB_NAME", content: "{{ postgresql_database }}" }
-  - { file: "MMW_DB_USER", content: "{{ postgresql_username }}" }
-  - { file: "MMW_DB_PASSWORD", content: "{{ postgresql_password }}" }
-  - { file: "MMW_DB_HOST", content: "{{ postgresql_host }}" }
-  - { file: "MMW_DB_PORT", content: "{{ postgresql_port }}" }
-  - { file: "MMW_TEST_DB_NAME", content: "{{ django_test_database }}" }
-  - { file: "MMW_STATSD_HOST", content: "{{ statsite_host }}" }
-  - { file: "MMW_CACHE_HOST", content: "{{ redis_host }}" }
-  - { file: "MMW_CACHE_PORT", content: "{{ redis_port }}" }
-  - { file: "DJANGO_SETTINGS_MODULE", content: "{{ django_settings_module }}" }
-  - { file: "DJANGO_STATIC_ROOT", content: "{{ app_static_root }}" }
-  - { file: "DJANGO_MEDIA_ROOT", content: "{{ app_media_root }}" }
-  - { file: "DJANGO_POSTGIS_VERSION", content: "{{ app_postgis_version }}" }
-  - { file: "DJANGO_SECRET_KEY", content: "{{ app_secret_key }}" }
-  - { file: "ITSI_CLIENT_ID", content: "{{ itsi_client_id }}" }
-  - { file: "ITSI_SECRET_KEY", content: "{{ itsi_secret_key }}" }
```

Comment:
The ITSI values did not get converted to the new data structure and seem to be the only ones left out.  Are they handled differently now? I don't see them addressed in other commits.